Detectors/TRD/workflow/src/TrackBasedCalibSpec.cxx

```diff
@@ -70,7 +70,10 @@ void TRDTrackBasedCalibDevice::run(ProcessingContext& pc)
   updateTimeDependentParams(pc); // Make sure this is called after recoData.collectData, which may load some conditions
   mCalibrator.setInput(recoData);
   mCalibrator.calculateAngResHistos();
+  mCalibrator.calculateGainCalibObjs();
```

Comment:
By default we definitely want the vDrift and ExB calibration during the synchronous reco, since they have an impact on the tracklets which are used for global tracking and the TPC calibration. During the asynchronous reco we would want both to be active.
The enable-trackbased-calib option should stay. And you should add another option to the trd-tracking-workflow named enable-gain-calib for example. This boolean flag (false by default) should be passed to the track based calib device which then either adds the gain calibration or not. 
So in case trackbased calibrations are enabled we will always have vD and ExB, but gain will be optional